src/app/resources/rate-resources/resources-rate.component.html

```diff
@@ -0,0 +1,45 @@
+<h1 i18n>Submit your rating</h1>
+<form [formGroup]="ratingForm" (ngSubmit)="onSubmit()">
+
+    <div hidden>
+        <mat-form-field>
+            <input matInput i18n-placeholder [(ngModel)]="this.id" type="text" formControlName="id">
+            <mat-error><planet-form-error-messages [control]="ratingForm.controls.id"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+    </div>
+
+    <div hidden>
+        <mat-form-field>
+            <input matInput i18n-placeholder [(ngModel)]="this.user" type="text" formControlName="user">
+            <mat-error><planet-form-error-messages [control]="ratingForm.controls.user"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+    </div>
+
```

Comment:
There are more extra line . Could you please remove all those ?

Reply:
@nirojdyola All removed https://github.com/ole-vi/planet/pull/270/commits/45ffaf3d071f9cb9676524bb17fa8f9484341306